src/data/serializer/Vector.js

```diff
@@ -200,6 +200,13 @@ Ext.define('GeoExt.data.serializer.Vector', {
                 var geometryType = geometry.getType();
                 var geojsonFeature = format.writeFeatureObject(feature);
 
+                // remove parent feature references as they break serialization
+                // later on
+                if (geojsonFeature.properties &&
+                        geojsonFeature.properties.parentFeature) {
```

Comment:
where exactly do we set these references in the first place?

Reply:
I didn't really have a deeper look for this. I've assumed that the link is between the label features and the actual measurement geometry, that's why I thought it would be best to destroy the link in the GeoJSON representation only.